demisto_sdk/commands/validate/validators/IF_validators/IF109_invalid_required_field.py

```diff
@@ -0,0 +1,114 @@
+            ) and item.git_status == GitStatuses.ADDED:
+                types_items.append(item.object_id)
+            elif isinstance(item, IncidentField) or isinstance(item, IndicatorField):
+                fields_items.append(item)
+
+        return [
+            ValidationResult(
+                validator=self,
+                message=error_res,
+                content_object=content_item,
+            )
+            for content_item in fields_items
+            if (error_res := self.is_invalid_required_field(content_item, types_items))
+        ]
+
+    @staticmethod
+    def is_invalid_required_field(
+        content_item: Union[IncidentField, IndicatorField], added_types: list[str]
+    ):
+        """
+        Get from the graph the actual fields for the given aliases
+
+        Args:
+            content_item (Union[IncidentField, IndicatorField]): The incident field or indicator field to check its required field.
+            added_types (list): incident or indicator types that were created/added in the same pull request.
+
+        Returns:
+            str or None: return none in case it's valid, the error message in case it's not.
+        """
+        # Required fields should not be associated to all
+        if content_item.required and content_item.associated_to_all:
+            return (
+                f"A required {'Incident' if isinstance(content_item, IncidentField) else 'Indicator'} Field"
+                f" should not be associated with all types."
+            )
+
+        if content_item.git_status == GitStatuses.MODIFIED:
+            old_file = cast(
+                Union[IncidentField, IndicatorField],
+                content_item.old_base_content_object,
+            )
+
+            # Required value for an already existing field cannot be changed
+            if content_item.required != old_file.required:
+                return (
+                    f"Required value should not be changed for an already existing"
+                    f" {'Incident' if isinstance(content_item, IncidentField) else 'Indicator'} Field."
+                )
+
+            # An already existing Incident/Indicator Type cannot be added to Incident/Indicator Field with required value true
+            if content_item.required and len(content_item.associated_types) > len(
+                old_file.associated_types
+            ):
+                new_types = list(
+                    filter(
+                        lambda x: x not in old_file.associated_types,
+                        content_item.associated_types,
+                    )
+                )
+                for new_type in new_types:
+                    if new_type not in added_types:
+                        return (
+                            f"An already existing Type like {new_type} cannot be added to an "
+                            f"{'Incident' if isinstance(content_item, IncidentField) else 'Indicator'} "
```

Comment:
```suggestion
                            f"{'Incident' if isinstance(content_item, IncidentField) else 'Indicator'} "
```
You can use  content_item.content_type